### DISCUSSION TITLE
Custom Estimate

### DISCUSSION THREAD

**Discussion by Dalkan75** [mclemente/healthEstimate#227] [General]
Hello,
I'm trying to create a new list of values. I tried to follow the advice, that is, to put the creature type value to identify the resource.
But unfortunately it doesn't work.
What am I doing wrong?
Is there a detailed tutorial for creating a list on objects other than PCs or NPCs?
Thank you and Happy New Year
PS: I use D&D5e
![immagine](https://github.com/mclemente/healthEstimate/assets/108159546/51e05d35-27cb-46fb-8b43-6802c7d80415)

**Comment by mclemente** [comment]
Is that a Custom Type? I'm assuming it is since Resource/Risorsa isn't on the base list. Try `actor.system.details.type.custom === "Risorsa"`.

Happy New Year!

**Comment by Dalkan75** [comment]
Thank you <3
it's work!
^__^

Have a nice day!
